platform/linuxbsd/export/export.cpp

```diff
@@ -31,27 +31,27 @@
 #include "export.h"
 
 #include "editor/editor_export.h"
-#include "platform/x11/logo.gen.h"
+#include "platform/linuxbsd/logo.gen.h"
 #include "scene/resources/texture.h"
 
-void register_x11_exporter() {
+void register_linuxbsd_exporter() {
 
 	Ref<EditorExportPlatformPC> platform;
 	platform.instance();
 
-	Ref<Image> img = memnew(Image(_x11_logo));
+	Ref<Image> img = memnew(Image(_linuxbsd_logo));
 	Ref<ImageTexture> logo;
 	logo.instance();
 	logo->create_from_image(img);
 	platform->set_logo(logo);
 	platform->set_name("Linux/X11");
 	platform->set_extension("x86");
 	platform->set_extension("x86_64", "binary_format/64_bits");
-	platform->set_release_32("linux_x11_32_release");
-	platform->set_debug_32("linux_x11_32_debug");
-	platform->set_release_64("linux_x11_64_release");
-	platform->set_debug_64("linux_x11_64_debug");
-	platform->set_os_name("X11");
+	platform->set_release_32("linuxbsd_32_release");
+	platform->set_debug_32("linuxbsd_32_debug");
+	platform->set_release_64("linuxbsd_64_release");
+	platform->set_debug_64("linuxbsd_64_debug");
+	platform->set_os_name("Unix");
```

Comment:
Just confirming. OS name is gonna be "Unix" and not "Linux/BSD"?

Reply:
No. `linuxbsd` on account of there already being a `unix` OS (that includes MacOS)

Reply:
So line 54 will be changed?

Reply:
Yes.